cypress/utils/navigate.ts

```diff
@@ -0,0 +1,16 @@
+export class Navigate {
+/*This function handles the webPopup to access the location. It will make it false*/
+    static HandleLocationPopup(){
+        Cypress.on('window:confirm',(message)=>{
+            if(message.includes('Know your location')){
+                return false;
+            }
+        })
```

Comment:
What is the use of this function ?

Reply:
@ajinkyaces  use of this function is for handling popup regarding granting location access. This function is used for cancelling the popup